parade/ddl.sql

```diff
@@ -8,7 +8,7 @@ CREATE TYPE task_status_code_value AS ENUM (
 );
 
 CREATE TABLE IF NOT EXISTS tasks (
-    id VARCHAR(64) NOT NULL PRIMARY KEY, -- nanoid
+    id VARCHAR(128) NOT NULL PRIMARY KEY, -- nanoid
```

Comment:
Note that this "128"\ appears in numerous other places.  And let's get rid of `nanoid` in the comment -- it never was.

Thinking about it, we should probably get rid of the length limitation.  Per https://www.postgresql.org/docs/12/datatype-character.html there is actually no advantage to this limitation, it only saves space when it truncates :scream: .

Reply:
Done, I removed the length limitation